internal/endpoints/github.go

```diff
@@ -24,11 +24,12 @@ func (eh *EndpointHandler) GithubWebhook(w http.ResponseWriter, r *http.Request)
 
 	services := config.Get().Services
 
-	if config.Get().Debug {
+	if config.Get().SkipWebhookValidation {
 		payload, err = ioutil.ReadAll(r.Body)
```

Comment:
Can you add a log message here (like in the gitlab code) telling us / warning us that we are skipping the validation?